lib/pages/feed_page.dart

```diff
@@ -130,7 +138,7 @@ class JobsPage extends StatelessWidget {
             );
           case ConnectionState.done:
             return (snapshot.hasError)
-                ? Text("${snapshot.error}")
+                ? Center(child: Text("${snapshot.error}"))
```

Comment:
why not Text(snaphot.error)? What is the goal of quotes?

Reply:
```snapshot.error``` is ```Object```. however, for some reason, the compiler isn't complaining, but at runtime I get ```TypeError```